content/200-concepts/100-components/02-prisma-client/080-null-and-undefined.mdx

```diff
@@ -174,13 +180,18 @@ const users = await prisma.user.findMany({
 // returns: []
 ```
 
-The query receives filters from a formData object, which includes an optional email property. In this instance, the value of the email property is `undefined`. When this query is run no data is returned.
+The query receives filters from a `formData` object, which includes an optional email property.
+
+- In this instance, the value of the email property is `undefined`. The query returns an empty list.
+- If the value of the email property is empty, instead of `undefined`, then the query returns all the users.
```

Comment:
A code snippet to illustrate this would've been very helpful, as I'm not sure I correctly understand what the difference should be.

Reply:
Thank you!

Reply:
So this is the same as above - we're talking about an empty string here. Do you think it would be sufficient to say:

```suggestion
- If the value of the email property is an empty string, instead of `undefined`, then the query returns all the users.
```